<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fichePOC</class>
 <widget class="QDialog" name="fichePOC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_nomPOC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nomPOC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_prenomPOC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenomPOC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_institutionPOC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_institutionPOC">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Institution</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mailPOC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mailPOC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_telephonePOC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_telephonePOC">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Telephone</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_FAXPOC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_FAXPOC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FAX </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fonctionPOC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>248</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fonction </string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_commentairePOC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>163</width>
     <height>53</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_commentairePOC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Commentaire</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_fonctionPOC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>248</y>
     <width>163</width>
     <height>53</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bloquedPOC">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>503</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Debloquer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimerPOC">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>503</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_commentairePOC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>393</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sauvegarderPOC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>503</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nomPOCControl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_prenomPOCControl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_institutionControl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_mailControl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_telephoneControl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_faxControl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_fonctionControl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>251</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_commentairePOCControl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>314</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
